src/backend/distributed/metadata/metadata_sync.c

```diff
@@ -386,8 +385,12 @@ MetadataCreateCommands(void)
 		ObjectAddressSet(tableAddress, RelationRelationId, relationId);
 		EnsureDependenciesExistOnAllNodes(&tableAddress);
 
-		metadataSnapshotCommandList = list_concat(metadataSnapshotCommandList,
-												  workerSequenceDDLCommands);
+		if (!PartitionTable(relationId))
```

Comment:
It is possible for a partition to have a different DEFAULT than its parent, so this can cause issues:
```sql
create sequence seq;
create table par (x text, s bigint default nextval('seq'), t timestamptz default now()) partition by range (t);
select create_distributed_table('par','x');

create sequence par1_seq;
create table par1 (x text, s bigint default nextval('par1_seq'), t timestamptz default now());
alter table par attach partition par1 for values from ('2021-05-31') to ('2021-06-01');
```